packages/origin-ui-core/src/components/Modal/WithdrawModal.tsx

```diff
@@ -0,0 +1,120 @@
+import { ProducingDevice } from '@energyweb/device-registry';
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    TextField
+} from '@material-ui/core';
+import { bigNumberify } from 'ethers/utils';
+import moment from 'moment';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { resyncCertificate, requestWithdrawCertificate } from '../../features/certificates';
+import { ICertificateViewItem } from '../../features/certificates/types';
+import { getExchangeClient } from '../../features/general/selectors';
+import { getUserOffchain } from '../../features/users/selectors';
+import { EnergyFormatter, formatDate } from '../../utils';
+
+interface IProps {
+    certificate: ICertificateViewItem;
+    producingDevice: ProducingDevice.Entity;
+    showModal: boolean;
+    callback: () => void;
+}
+
+const DEFAULT_ENERGY_IN_BASE_UNIT = bigNumberify(1);
```

Comment:
This should be changed into a .env variable as mentioned here https://github.com/energywebfoundation/origin/issues/1081